stdlib/2.7/__builtin__.pyi

```diff
@@ -529,7 +529,7 @@ class dict(MutableMapping[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, iterable: Iterable[Tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
+    def __init__(self, iterable: Iterable[Union[Tuple[_KT, _VT], Sequence[_VT]]], **kwargs: _VT) -> None: ...
```

Comment:
Mypy special cases `dict(x='y')` to do the right thing. We could do the same thing for a sequence argument but as this is a less frequent use case it's less clear whether it's worth the complexity and effort.